File: jour_26/jour26.py
class Etudiant:
    def __init__(self, nom, age, note):
        self.nom = nom
        self.age = age
        self.note = note
# Création et remplissage
nom = input("Nom : ")
age = int(input("Âge : "))
note = float(input("Note : "))
e1 = Etudiant(nom, age, note)
print(f"\nL'étudiant {e1.nom} a {e1.age} ans et une note de {e1.note}/20")

File: jour_26/jour26.c
#include <stdio.h>

// Définition de la structure
struct Etudiant {
    char nom[50];
    int age;
    float note;
};

int main() {
    struct Etudiant e1;

    // Remplissage des données
    printf("Nom de l'étudiant : ");
    scanf("%s", e1.nom);
    printf("Âge : ");
    scanf("%d", &e1.age);
    printf("Note : ");
    scanf("%f", &e1.note);

    printf("\nL'étudiant %s a %d ans et une note de %.2f/20\n", e1.nom, e1.age, e1.note);

    return 0;
}